src/Mod/Measure/App/Measurement.h

```diff
@@ -39,7 +39,17 @@ namespace Measure
  enum MeasureType {
```

Comment:
Can you add it to your long-term list to migrate this to an `enum class`?

Reply:
I had a look and this enum is used in techdraws in ways that I don't fully understand. It seems to be associated to a PropertyEnumeration. So I am not sure how to handle this.

Reply:
But yes I can add this to my long term list

Reply:
After a second look, I think that TechDraw is actually not using this MeasureType enum.

Reply:
So this is ok I changed it to enum class